src/bot/Bot.ts

```diff
@@ -11,6 +13,31 @@ import { WakatimeContext } from './Context.js';
 import { parseSchedule } from './middleware/parseSchedule.js';
 import { sendLeaderboard } from './sendLeaderboard.js';
 
+// config Limit Request Users
+
+const redis = new Redis();
+
+Bot.use(
+  limit({
+    // Allow only 3 messages to be handled every 2 seconds.
+    timeFrame: 2000,
+
+    limit: 3,
+
+    // "MEMORY_STORE" is the default value. If you do not want to use Redis, do not pass storageClient at all.
+    storageClient: redis,
```

Comment:
Let's utilize "MEMORY STORE" because it's easier to operate without difficulties and configuration, and we don't have many users to consider.